Tests/Marketplace/approved_tags.json

```diff
@@ -31,6 +31,9 @@
 		"Spam",
 		"TIM",
 		"Threat Intelligence",
-		"VPN"
+		"VPN",
+		"Provisioning",
+		"Employee",
+		"ILM"
```

Comment:
no need to add these new tags. we need to remove them from the IAM pack itself

Reply:
we don't want them anymore? not even ILM?

Reply:
we don't. We have IAM , it is enough